src/functionalTest/java/uk/gov/hmcts/darts/usermanagement/UserManagementFunctionalTest.java

```diff
@@ -203,25 +236,36 @@ private Response createUser() {
                            "description": "A temporary user created by functional test"
                       }
                       """)
-            .post()
+            .post(getUri(ADMIN_USERS_BASE_PATH))
             .thenReturn();
 
         assertEquals(201, response.getStatusCode());
 
         return response;
     }
 
-    @Test
-    void shouldGetSecurityGroups() throws JsonProcessingException {
+    private Response createUserWithSecurityGroups() {
         Response response = buildRequestWithExternalGlobalAccessAuth()
-            .baseUri(getUri("/admin/security-groups"))
             .contentType(ContentType.JSON)
-            .get()
+            .body("""
+                      {
+                           "full_name": "James Smith",
```

Comment:
i am currently looking to make a proposal that we prefix all strings with func-. We can then have a general rule for devs not to create records with a prefix of func-. This should act as a belt and braces approach to stabilising our tests.